FrontEnd/src/components/ProfileDetail/DetailedInfo/CompanyDescription.jsx

```diff
@@ -10,10 +10,10 @@ function CompanyDescription ({ data }) {
     return (
         <div className={classes['company-description-block']}>
             <Company data={data} />
-            <Startup data={data} />
-            <ProductsServices data={data} />
-            <Logistics />
-            <Cooperation />
+            {data.is_startup ? <Startup data={data} /> : null}
```

Comment:
It could be simplified `{data.is_startup && <Startup data={data} />}`

Reply:
Done